x-pack/plugins/monitoring/dev_docs/reference/data_collection_modes.md

```diff
@@ -35,11 +35,11 @@ subgraph Kibana
   Collectors-.->|poll|Metrics[OS, Cgroups, etc]
   Collectors-.->|update|OpsMetricsObservable
   BulkUploader-.->|observe|OpsMetricsObservable
-  
-  click Collectors "https://github.com/elastic/kibana/tree/main/src/core/server/metrics/collectors"
-  click OpsMetricsObservable "https://github.com/elastic/kibana/blob/92a8636f0ff63ab072527574e96e6616327b2ea4/src/core/server/metrics/metrics_service.ts#L32"
+
+  click Collectors "https://github.com/elastic/kibana/tree/main/src/core/packages/core/server/metrics/collectors"
+  click OpsMetricsObservable "https://github.com/elastic/kibana/blob/92a8636f0ff63ab072527574e96e6616327b2ea4/src/core/packages/core/server/metrics/metrics_service.ts#L32"
```

Comment:
```suggestion
  click Collectors "https://github.com/elastic/kibana/tree/main/src/core/server/metrics/collectors"
  click OpsMetricsObservable "https://github.com/elastic/kibana/blob/92a8636f0ff63ab072527574e96e6616327b2ea4/src/core/server/metrics/metrics_service.ts#L32"
```

Reply:
I'm actually moving the `server` folder, so I think the 1st line is correct? I'll revert changes in the second.

Reply:
My point is that `src/core/metrics` don't exist since we moved the service to a package. But we forgot to update this reference.

I'd prefer to update these already broken URLs in a separate PR to avoid making too much noise in this important PR.